test/components/TabbedNavTest.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert')
+const React = require('react')
+const ReactDOM = require('react-dom')
+const TestUtils = require('react-addons-test-utils')
+const TabbedNav = require('../../src/components/TabbedNav')
+
+describe('TabbedNav', () => {
+  let renderedDOM
+
+  before(() => {
+    const component = TestUtils.renderIntoDocument(<TabbedNav />)
+    renderedDOM = () => ReactDOM.findDOMNode(component)
+  })
+
+  it('has three links', () => {
+    const notificationsLink = renderedDOM().querySelectorAll('notifications-link')
+    const filtersLink = renderedDOM().querySelectorAll('filters-link')
+    const authLink = renderedDOM().querySelectorAll('auth-link')
+
+    assert(notificationsLink)
```

Comment:
Same problem as my comment above. `querySelectorAll` returns an array, and an empty array is always truthy. You're going to get nothing but empty arrays back when your selector is `'auth-link'`, for example, because that's not a CSS selector. I think you want `.auth-link` to find the element with that class, or `#auth-link` to find the element with that ID.

![screen shot 2016-09-22 at 2 37 53 pm](https://cloud.githubusercontent.com/assets/82317/18763274/2dc749ea-80d2-11e6-8f1b-9b9da7b2a901.png)

I'd use `querySelector` which will return the first matching element or null, and a null is not truthy so the `assert` will fail as desired.